### ISSUE TITLE
QA master tarball fails integrity check for .htaccess file

### ISSUE THREAD

**Issue by individual-it** [owncloud/core#34787]
### Steps to reproduce
1. download and install master QA tarball: https://download.owncloud.org/community/owncloud-daily-master-qa.tar.bz2
2. run integrity check: `sudo -u www-data ./occ integrity:check-core`

### Expected behaviour
no output integrity check should pass

### Actual behaviour
```
  - core:
    - INVALID_HASH:
      - .htaccess:
        - expected: 6c0fa034eb1fdd485e7557dabeb889b5a5db40a75800cb046f817e45974c35ca1a214d4106a864e99eade58b66b2a9c2aeb076cc7cf81c130cb167b45872db52
        - current: beea5f10df3f26a6700fa280b2919bdc51aa7c988396ab0bd6a0638c489fb3cc4aeb77d99251a59516c3bd44cb22fe319e9b580eb3abecb20ea8b6672c3f4fd5
```

This leads to issue like: https://github.com/owncloud/password_policy/issues/186 and https://github.com/owncloud/user_management/issues/167

**Comment by PVince81** [comment]
@individual-it diff between the htaccess from tarball and the one from core ? and the one from `make dist-qa` ?

**Comment by individual-it** [comment]
fixed by  #34825
